src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/parser/format/MetricJsonFormat.kt

```diff
@@ -0,0 +1,10 @@
+package com.atlassian.performance.tools.jiraactions.api.parser.format
+
+import com.atlassian.performance.tools.jiraactions.api.ActionMetric
+import javax.json.JsonObject
+
+interface MetricJsonFormat {
+    fun serialize(actionMetric: ActionMetric): JsonObject
```

Comment:
This might look problematic, because in the future we might want o use more efficient future formats (like binary Protobuf), which are not JSON.
But we learned how to expand interfaces without breaking compatibility, so it's perfectly fine to leave this as is.